test_res.txt

```diff
@@ -0,0 +1,528 @@
+ok  	github.com/moira-alert/moira	(cached)
```

Comment:
hmmm, added into gitignore?

Reply:
I acidently added this file to git. Deleted :ok_hand: